homeassistant/components/ring/__init__.py

```diff
@@ -32,6 +44,7 @@ def setup(hass, config):
     conf = config[DOMAIN]
     username = conf[CONF_USERNAME]
     password = conf[CONF_PASSWORD]
+    scan_interval = conf.get(CONF_SCAN_INTERVAL)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
done :)